src/dynamic_boundary_conditions/hirds_gauges.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Dec 23 16:06:53 2021.
+
+@author: pkh35
+"""
+
+import requests
+from requests.structures import CaseInsensitiveDict
+import pandas as pd
+from shapely.geometry import Point
+import geopandas as gpd
+import sqlalchemy
+from geoalchemy2 import Geometry
+import geopandas
+import numpy as np
+
+
+def get_hirds_gauges_data() -> pd.DataFrame:
+    """Get gauges information from the hirds website using curl command."""
+    url = "https://api.niwa.co.nz/hirds/sites"
+    headers = CaseInsensitiveDict()
+    headers["Connection"] = "keep-alive"
+    headers["sec-ch-ua"] = '"" Not A;Brand";v="99", "Chromium";v="96", "Google Chrome";v="96""'
+    headers["Accept"] = "application/json, text/plain, */*"
+    headers["sec-ch-ua-mobile"] = "?0"
+    headers["User-Agent"] = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko)\
+        Chrome/96.0.4664.45 Safari/537.36"
+    headers["sec-ch-ua-platform"] = "Windows"
+    headers["Origin"] = "https://hirds.niwa.co.nz"
+    headers["Sec-Fetch-Site"] = "same-site"
+    headers["Sec-Fetch-Mode"] = "cors"
+    headers["Sec-Fetch-Dest"] = "empty"
+    headers["Referer"] = "https://hirds.niwa.co.nz/"
+    headers["Accept-Language"] = "en-GB,en-US;q=0.9,en;q=0.8"
+
+    response = requests.get(url, headers=headers)
```

Comment:
If the response fails (such as if hirds website is down) then you need an error checking

Reply:
yup, you are right, I'll add that functionality to it. Thank you :)

Reply:
Raised issue #23 to solve this